### PR TITLE
Add schema for /states

### LINKED ISSUE #9: Endpoint.StateEndpoint
Related to #4

### PR DESCRIPTION
**Pull request by ian952**
Fixes #9

### CODE REVIEW

File: schemas/components/anomalyDetectorState.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      type: number
+    fixableViolatedGoals:
+      nullable: true
+      type: array
+      items:
+        type: string
+    unfixableViolatedGoals:
+      nullable: true
+      type: array
+      items:
```
Comment (Will-Lo) [inline schemas/components/anomalyDetectorState.yaml]: Trying to understand nullable here, is it every field that appears in verbose nullable?

**Review by ekicode — APPROVED** [review]
LGTM 👍 verified with redoc-cli.

(FYI: haven't cross-checked the parameters/responses against cruise-control codebase).

File: schemas/components/anomalyDetectorState.yaml (hunk 1 above)
Comment (ian952) [reply schemas/components/anomalyDetectorState.yaml]: Yes. Fields can also be nullable if it is only added to the JSON depending on some internal state, usually involving some switch statement.

Comment (Will-Lo) [reply schemas/components/anomalyDetectorState.yaml]: LGTM then, I'll relay this to Kun as well

**Comment by Will-Lo** [comment]
Will approve when conflicts are fixed

**Pull request merged**